src/main/java/net/snowflake/client/core/HttpUtil.java

```diff
@@ -318,8 +344,15 @@ public static CloseableHttpClient buildHttpClient(
         logger.error(errors.toString(), true);
         throw new RuntimeException(err); // rethrow the exception
       }
+    } else if (key != null) {
+      logger.debug("Omitting trust manager instantiation as OCSP mode is set to {}", key.getOcspMode());
+    } else {
+      logger.debug("Omitting trust manager instantiation as configuration is null");
     }
     try {
+      logger.debug("Registering https connection socket factory with socks proxy disabled: {}", socksProxyDisabled);
+      logger.debug("Registering http connection socket factory");
```

Comment:
do we need both debug logs?

Reply:
I have consolidated this and made single debug statement.